tests/src/CoreMangLib/system/delegate/VSD/VSD_ResolveWorker.cs

```diff
@@ -0,0 +1,55 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
you may clean out the test to remove unncessary stuff. Also maybe openDelegate might be a better name of this test